_DeploymentAndDistroPackaging/ansible/doc/development.md

```diff
@@ -1,8 +1,15 @@
 # Development Mode
-This playbooks installs ciao from the latest ciao packaged for your distro.
+This playbooks are though to install ciao on production environments from the
+latest packages availabe for each supported distro.
 
-If you want to try the latest features, there is a way to use this playbook to
-deploy ciao from github.com/01org/ciao master branch source code.
+above you can finde some tips that can be useful if you plan to try new features
```

Comment:
This sentence is a bit confusing as well.  How about

'However, there are some options that can be specified when running the playbooks to alter this default behaviour.  These options are useful during development and are described in more detail below.'